shards/gfx/gltf/gltf.cpp

```diff
@@ -674,6 +674,57 @@ struct Loader {
   }
 };
 
+// Helper function to convert tinygltf matrix to linalg matrix
+linalg::mat<double, 4, 4> convertMatrix(const std::vector<double> &m) {
```

Comment:
Can you use `convertNodeTransform` that already exists.
TRS has a toMat conversion in case you want matrix instead

Reply:
nice ok